src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -12,5 +12,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_START_TIME = new Prefix("s/");
+    public static final Prefix PREFIX_EVENT = new Prefix("ev/");
```

Comment:
Should we consider make link to event a separate command instead of adding more prefixes?